lib/openpgp_crypt_symmetric.php

```diff
@@ -188,34 +191,41 @@ public static function getCipher($algo) {
         }
         break;
       case 4:
-        $cipher = new Crypt_Blowfish(Crypt_Blowfish::MODE_CFB);
+        $cipher = new Crypt_Blowfish('cfb');
+        $cipher->setIV(str_repeat(pack('x'), 8));
         $key_bytes = 16;
         $key_block_bytes = 8;
         break;
       case 7:
-        $cipher = new Crypt_AES(Crypt_AES::MODE_CFB);
+        $cipher = new Crypt_AES('cfb');
         $cipher->setKeyLength(128);
+        $cipher->setIV(str_repeat(pack('x'), 16));
         break;
       case 8:
-        $cipher = new Crypt_AES(Crypt_AES::MODE_CFB);
+        $cipher = new Crypt_AES('cfb');
         $cipher->setKeyLength(192);
+        $cipher->setIV(str_repeat(pack('x'), 16));
         break;
       case 9:
-        $cipher = new Crypt_AES(Crypt_AES::MODE_CFB);
+        $cipher = new Crypt_AES('cfb');
         $cipher->setKeyLength(256);
+        $cipher->setIV(str_repeat(pack('x'), 16));
         break;
       case 10:
-        $cipher = new Crypt_Twofish(Crypt_Twofish::MODE_CFB);
+        $cipher = new Crypt_Twofish('cfb');
+        $cipher->setIV(str_repeat(pack('x'), 16));
         if(method_exists($cipher, 'setKeyLength')) {
-          $cipher->setKeyLength(256);
+          $cipher->setKeyLength(128);
         } else {
           $cipher = NULL;
         }
         break;
     }
     if(!$cipher) return array(NULL, NULL, NULL); // Unsupported cipher
-    if(!isset($key_bytes)) $key_bytes = isset($cipher->key_size)?$cipher->key_size:$cipher->key_length;
-    if(!isset($key_block_bytes)) $key_block_bytes = $cipher->block_size;
+
+
+    if(!isset($key_bytes)) $key_bytes = $cipher->getKeyLength() >> 3;
```

Comment:
instead of checking the existence of key_size , I've instead make the different wrappers implement the same set of method